content/spin/v2/sqlite-api-guide.md

```diff
@@ -143,27 +143,30 @@ const json = JSON.stringify(result.rows);
 
 {{ startTab "Python"}}
 
-> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/spin_sqlite.html)
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/sqlite.html)
 
-To use SQLite functions, use the `spin_sqlite` module in the Python SDK. The [`sqlite_open`](https://fermyon.github.io/spin-python-sdk/spin_sqlite.html#spin_sdk.spin_sqlite.sqlite_open) and [`sqlite_open_default`](https://fermyon.github.io/spin-python-sdk/spin_sqlite.html#spin_sdk.spin_sqlite.sqlite_open_default) functions return a [connection object](https://fermyon.github.io/spin-python-sdk/spin_sqlite.html#spin_sdk.spin_sqlite.SqliteConnection). The connection object provides the [`execute` method](https://fermyon.github.io/spin-python-sdk/spin_sqlite.html#spin_sdk.spin_sqlite.SqliteConnection.execute) as described above. For example:
+To use SQLite functions, use the `sqlite` module in the Python SDK. The [`sqlite_open`](https://fermyon.github.io/spin-python-sdk/sqlite.html#spin_sdk.sqlite.open) and [`sqlite_open_default`](https://fermyon.github.io/spin-python-sdk/sqlite.html#spin_sdk.sqlite.open_default) functions return a [connection object](https://fermyon.github.io/spin-python-sdk/wit/imports/sqlite.html#spin_sdk.wit.imports.sqlite.Connection). The connection object provides the [`execute` method](https://fermyon.github.io/spin-python-sdk/wit/imports/sqlite.html#spin_sdk.wit.imports.sqlite.Connection.execute) as described above. For example:
 
 ```python
-from spin_http import Response
-from spin_sqlite import sqlite_open_default
-
-def handle_request(request):
-    conn = sqlite_open_default()
-    result = conn.execute("SELECT * FROM todos WHERE id > (?);", [1])
-    rows = result.rows()
-    return Response(200,
-                    {"content-type": "application/json"},
-                    bytes(str(rows), "utf-8"))
+from spin_sdk.http import simple
+from spin_sdk.http.simple import Request, Response
+from spin_sdk import sqlite
+from spin_sdk.wit.imports.sqlite import ValueInteger
+
+class IncomingHandler(simple.IncomingHandler):
+    def handle_request(self, request: Request) -> Response:
+        with sqlite.open_default() as db:
+            print(db.execute("SELECT * FROM todos WHERE id > (?);", [ValueInteger(1)]))
+        
+        return Response(
+            200,
+            {"content-type": "text/plain"},
+            bytes("Hello from Python!", "utf-8")
```

Comment:
Returning the rows in the response felt way more natural.  Dumping the query results to stdout and returning a "hello world" page feels a bit silly.